include/picongpu/param/pusher.param

```diff
@@ -28,6 +28,31 @@
 
 namespace picongpu
 {
+    struct particlePusherAccelerationParam
+    {
+        /** Define strength of constant and homogeneous accelerating
+         *  electric field in SI per dimension.
+         */
+
+        /** unit: Volt / meter */
+        static constexpr float_64 AMPLITUDEx_SI= 0.0;
```

Comment:
style: spaces around `=` pls (3x in this file)

Reply:
I will do so, but in a new PR as this is already merged. Or can I push to this PR again?

Reply:
No, just do a follow-up PR.